load_test.py

```diff
@@ -116,13 +116,15 @@ def main(args):
     """Load test CLI entrypoint."""
     args = utils.parse_args(args)
 
+    mp_mgr = mp.Manager()
+
     mp_ctx = mp.get_context("spawn")
```

Comment:
You could also invoke a Manager class from this context-based proxy object. `mp.get_context("spawn").Manager()` that should also work.

Reply:
thanks Nikhil, I wanted to have a way to link the manager and the context, but I couldn't find it when I first tried.
This one works 👍🏻